Project/src/MakeCall/MakeCall.js

```diff
@@ -324,6 +330,34 @@ export default class MakeCall extends React.Component {
 
         return callOptions;
     }
+
+    async runPreCallDiagnostics() {
+        try {
+            this.setState({showPreCallDiagnostcisResults: false});
+            this.setState({isPreCallDiagnosticsCall: true});
+            this.setState({preCallDiagnosticsResults: {}});

```

Comment:
you could change to just one `this.setState`

like this:


```
this.setState({
    showPreCallDiagnostics: false,
    isPreCallDiagnosticsCall: true,
    preCallDiagnosticsResults: {}
})
```

Reply:
Done!